core/src/main/java/org/testcontainers/utility/CommandLine.java

```diff
@@ -57,15 +59,19 @@ public static String runShellCommand(String... command) {
      */
     public static boolean executableExists(String executable) {
 
+        String winExecutable = executable + ".exe";
+
         // First check if we've been given the full path already
         File directFile = new File(executable);
-        if (directFile.exists() && directFile.canExecute()) {
+        File windowsExeFile = new File(winExecutable);
```

Comment:
Ah, what I've suggested won't work if we need to check for _both_ names on Windows. Still, I think something similar might be easier to read. What do you think?